app/models/concerns/public_body_derived_fields.rb

```diff
@@ -7,13 +7,13 @@ module PublicBodyDerivedFields
     before_save :set_first_letter
 
     # When name or short name is changed, also change the url name
-    def short_name=(short_name)
-      super
+    def short_name=(value)
+      write_attribute("short_name", value)
```

Comment:
Lets link to the globalize source in the commit message to show where this was taken from.

Use symbol keys for the attr names if possible.